.github/workflows/ci.yml

```diff
@@ -72,7 +72,7 @@ jobs:
             path: html
 
   linux-testing:
-    name: ${{ matrix.os }}_${{ matrix.cxx_compiler }}_backend=${{ matrix.backend }}_device=${{ matrix.device_type }}_cxx${{ matrix.std }}_${{ matrix.build_type }}
+    name: be=${{ matrix.backend }},device=${{ matrix.device_type }},cmplr=${{ matrix.cxx_compiler }},os=${{ matrix.os }},std=${{ matrix.std }},cfg=${{ matrix.build_type }}
```

Comment:
I slightly prefer to have `backend` instead of `be`. If people think otherwise, I am ready to approve it as well. Overall - big improvement. Please also rebase it on the last state of the main branch as Dan suggested

Reply:
I do not have strong opinion on that as well.
@danhoeflinger what do you think?

Reply:
I think `be` is clear enough, especially after people get used to it.  I think characters come at a high cost here, so I'd keep it as it is if it were up to me.

Reply:
@danhoeflinger It took me several to figure it out what be means (I had several variants). But I don't want to insist

Reply:
Is there any *visible* place we can place a legend for the checks?  I'm not seeing a great place, but I don't know much about options available to admins for the repo.
I don't have a strong opinion here, `bknd` is a more explicit middleground but still kicks out the compiler from the "checks" view.

Reply:
I checked available tools which I have for my own forks. Looks like there are no such options.

I've seen another approach which solves that problem: create multiple workflow files, thus moving one parameter to a workflow name. This could be a backend for oneDPL. This will result in the following:

-  "Checks" tab will have multiple workflows
- "Show all checks" panel in the "Conversation" tab will list jobs like

```
dpcpp backend / device=CPU,cmplr=icpx,os=ubuntu-20.04,std=17,cfg=release
...
tbb backend / cmplr=g++,os=ubuntu-18.04,std=17,cfg=release
...
```
I think the current readability is not that bad for that change...

Reply:
Thanks for looking at that.  I would vote to merge as it is and if it creates confusion, we could explore something like that in the future.